conf/application.conf

```diff
@@ -233,6 +238,10 @@ feature-toggles {
   log-account-creation-json {
     enabled = false
   }
+
+  dwp-uc-check {
```

Comment:
do you need this flag?

Reply:
I think we will in the future? Maybe its in the wrong repo?

Reply:
this'd go in the BE where the eligibility check logic sits

Reply:
ok removed :)